AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/allocate-hearings/allocate-hearings.component.spec.ts

```diff
@@ -454,4 +454,47 @@ describe('AllocateHearingsComponent', () => {
             expect(clockIcon).toBeFalsy();
         });
     });
+
+    describe('Clear Hearing Updated Message', () => {
+        let HEARING_HAVE_BEEN_UPDATED: string;
+        let OTHER_MESSAGES: string;
+        beforeAll(() => {
```

Comment:
`beforeAll()` shouldn't be needed here, think you can change the `let`s to

```
const HEARING_HAVE_BEEN_UPDATED: string = 'Hearings have been updated.';
const OTHER_MESSAGES: string = 'Other messages';
```

and then remove the `beforeAll`

Reply:
const added and beforeAll() removed